src/interactive/README.md

```diff
@@ -104,3 +113,94 @@ Only clears the appended suggestion when you restart the interactive shell
 ```
 
 The color option will be saved.
+
+# Intelligent AZ Interactive (Revolutionary Change)
+## Recommendation
+
+We have integrated the cli recommendation to make the completion ability more intelligent and provide the scenario completion. This is a revolutionary change. Users can enable or disable the recommendation feature by running the following commands:
+
+```
+$ az config set interactive.enable_recommender=True # Default, try the new recommendation feature
+$ az config set interactive.enable_recommender=False # Disable the recommendation feature
+```
+Here is a demo video demonstrating the main features of the new az interactive:
+
+[![intelligent_az_interactive](docs/intelligent_az_interactive.png)](docs/intelligent_az_interactive.mp4 "intelligent_az_interactive")
+
+### Command Recommendation
+
+Command recommendation uses an intelligent algorithm to suggest the most relevant and frequently used command for the user based on their historical command inputs. By analyzing the user's input history and matching it with the highest relevant commands, Intelligent Az Interactive can assist Azure CLI beginners in making quick and informed decisions, saving them time and effort.
+
+The command recommendation feature can also be a good way to broaden users' knowledge boundaries and help them recognize and learn commands that are related to their common commands but never used before.
+
+When you don't know what to do next, you can enter Space, and it will recommend the next command which is most used in other similar sessions.
+
+<img align=center src="docs/command_recommendation.gif" width = "70%" alt="command_recommendation">
+
+### Scenario Identification
+
+Scenario identification is another powerful feature that automatically recognizes the user's current scenario and recommends a set of commands that are most relevant to the user's needs. This feature is based on our extensive database of over 600+ business scenarios, which cover a wide range of use cases across various scenes. By analyzing the user's input history, Intelligent Az Interactive can identify the relevant scenario and suggest the most appropriate command set, helping new CLI users to quickly accomplish their tasks with confidence and ease.
+
+You can enter Space and select the scenario you want to use. It will automatically execute the commands in the scenario and help you complete the task.
```

Comment:
```suggestion
**Usage**: You can enter `Space` and select the scenario *(the options with prefix `command set`)* you want to use. It will automatically execute the commands in the scenario and help you complete the task.
```